content/en/docs/ambient/usage/l7-features/index.md

```diff
@@ -6,72 +6,90 @@ owner: istio/wg-networking-maintainers
+Without a waypoint installed, you can only use [Layer 4 security policies](/docs/ambient/usage/l4-policy/). By adding a waypoint, you gain access to the following policies:
 
 |  Name  | Feature Status | Attachment |
 | --- | --- | --- |
-| `AuthorizationPolicy` (including L7 features) | Beta | `targetRefs` |
-| `RequestAuthentication` | Beta | `targetRefs` |
+| [`AuthorizationPolicy`](/docs/reference/config/security/authorization-policy/) (including L7 features) | Beta | `targetRefs` |
+| [`RequestAuthentication`](/docs/reference/config/security/request_authentication/) | Beta | `targetRefs` |
+
+### Considerations for authorization policies {#considerations}
+
+In ambient mode, authorization policies can either be *targeted* (for ztunnel enforcement) or *attached* (for waypoint enforcement). For an authorization policy to be attached to a waypoint it must have a `targetRef` which refers to that waypoint.
+
+The ztunnel cannot enforce L7 policies. If a policy with rules matching L7 attributes is targeted at a ztunnel, it will fail safe by becoming a `DENY` policy.
+
+See [the L4 policy guide](/docs/ambient/usage/l4-policy/) for more information, including when to attach policies to waypoints for TCP-only use cases.
 
 ## Observability
 
 The [full set of Istio traffic metrics](/docs/reference/config/metrics/) are exported by a waypoint proxy.
 
 ## Extension
 
-As the waypoint proxy is a deployment of {{< gloss >}}Envoy{{< /gloss >}}, the extension mechanisms that are available for Envoy in {{< gloss >}}sidecar{{< /gloss >}} mode are also available to waypoint proxies.
+As the waypoint proxy is a deployment of {{< gloss >}}Envoy{{< /gloss >}}, the extension mechanisms that are available for Envoy in {{< gloss "sidecar">}}sidecar mode{{< /gloss >}} are also available to waypoint proxies.
 
 |  Name  | Feature Status | Attachment |
 | --- | --- | --- |
-| `WasmPlugin` | Alpha | `targetRefs` |
+| `WasmPlugin` † | Alpha | `targetRefs` |
 | `EnvoyFilter` | Alpha | `targetRefs` |
 
-Read more on how to extend waypoints with Wasm plugins [here](/docs/ambient/usage/extend-waypoint-wasm/).
+† [Read more on how to extend waypoints with WebAssembly plugins](/docs/ambient/usage/extend-waypoint-wasm/).
 
-## Targeting policies or routing rules
+Extension configurations are considered policy by the Gateway API definition.
+
+## Scoping routes or policies
+
+A route or policy can be scoped to apply to all traffic traversing a waypoint proxy, or only specific services.
 
 ### Attach to the entire waypoint proxy
 
-To attach a policy or routing rule to the entire waypoint — so that it applies to all traffic enrolled to use it — set `Gateway` as the `parentRefs` or `targetRefs` value, depending on the type.
+To attach a route or a policy to the entire waypoint — so that it applies to all traffic enrolled to use it — set `Gateway` as the `parentRefs` or `targetRefs` value, depending on the type.
```

Comment:
I think it would be helpful to include _when/why_ you would want to do this here - for policies with L7 attributes dependent on the identity of the calling service (as mentioned in the L4 docs under "Choosing enforcement points when waypoints are introduced " - any other specific use cases?) and mention implications for scope and which services are fronted by the waypoint (single service account vs all services in a namespace).

Reply:
I believe we intend to write such s guide but this doc was scoped down to be useful until that happened.

I kind of stole it off Ian 😁